rescript/citations.bib

```diff
@@ -157,3 +157,21 @@ @article{schoch2020
   pmcid = {PMC7408187},
   pmid = {32761142}
 }
+
+@article{nilsson2019unite,
+  author = {Nilsson, Rolf Henrik and Larsson, Karl-Henrik and Taylor, Andy F S and Bengtsson-Palme, Johan and Jeppesen, Thomas S and Schigel, Dmitry and Kennedy, Peter and Picard, Kathryn and Glöckner, Frank Oliver and Tedersoo, Leho and Saar, Irja and Kõljalg, Urmas and Abarenkov, Kessy},
+  title = "{The UNITE database for molecular identification of fungi: handling dark taxa and parallel taxonomic classifications}",
+  journal = {Nucleic Acids Research},
+  volume = {47},
+  number = {D1},
+  pages = {D259-D264},
+  year = {2018},
+  month = {10},
+  abstract = "{UNITE (https://unite.ut.ee/) is a web-based database and sequence management environment for the molecular identification of fungi. It targets the formal fungal barcode—the nuclear ribosomal internal transcribed spacer (ITS) region—and offers all ∼1 000 000 public fungal ITS sequences for reference. These are clustered into ∼459 000 species hypotheses and assigned digital object identifiers (DOIs) to promote unambiguous reference across studies. In-house and web-based third-party sequence curation and annotation have resulted in more than 275 000 improvements to the data over the past 15 years. UNITE serves as a data provider for a range of metabarcoding software pipelines and regularly exchanges data with all major fungal sequence databases and other community resources. Recent improvements include redesigned handling of unclassifiable species hypotheses, integration with the taxonomic backbone of the Global Biodiversity Information Facility, and support for an unlimited number of parallel taxonomic classification systems.}",
```

Comment:
this line can be deleted as this info is not used by QIIME 2 (and would be better removed for brevity)